src/mapblock.h

```diff
@@ -513,6 +512,16 @@ class MapBlock /*: public NodeContainer*/
 	void serializeNetworkSpecific(std::ostream &os, u16 net_proto_version);
 	void deSerializeNetworkSpecific(std::istream &is);
 
+	// Use only when the MapBlock is help in scope outside.
```

Comment:
Might I suggest moving the two function to line 356 where the other getNode function are.
Specifically, just below setNodeNoCheck.
Maybe with `// non-validating variants of the above` instead for the comment.
____

@stan5: I believe it's something along the line of `It should only be use when the MapBlock has been loaded by another function in the same scope/function`.